data/ir-ads

```diff
@@ -0,0 +1,355 @@
+# List of ads and tracking company in Iran.
+# Thanks to PersianBlocker project
+# https://raw.githubusercontent.com/MasterKia/PersianBlocker/main/PersianBlockerHosts.txt
+
+#-------------------------v Persian Ads/Trackers/Annoyances Filters v-----------------------#
+# >> A <<
+analyt.ir
+
+scripts-ads.s3.ir-thr-at1.arvanstorage.com
+error-tracking.arvancloud.com
+
+adexo.ir
+adskav.com
+advn.ir
+
+amaroid.net
+
+analytics.aasaam.com
+analytics-2.aasaam.com
+analytics-3.aasaam.com
+
+adwised.com
+adwisedfs.com
+adwised.azureedge.net
+# TODO: Remove adwised.azureedge.net, but navatv.com and baran-music.com still links to it
+
+adivery.com
+adnegah.net
+adpulse.ir
+adsima.net
+adtrace.io
+affili.ir
+amarfa.ir
+ayyaar.ir
+# ^ Used by irna.ir
```

Comment:
What does the phrase `used by` means? Does it mean this site is also necessary for the `irna.ir` to function normally?

Reply:
It means these ad websites are used in irna.ir (Iran largest news agency). Irna is working great after blocking them.